### PR TITLE
feature-set-tailwind-config

### PR DESCRIPTION
**Pull request by star82620**
- tailwind config 
- style guild line 設定

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+h5 {
+  font-size: 18px;
+  font-weight: 400;
+  line-height: 1.4;
+}
+h6 {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 1.4;
+}
```
Comment (seonkuraito) [inline styles/globals.css:39]: 1. box-sizing
2. a: cursor, text-decoration
3. html, body: width & height / flex & flex-direction (optional)

File: pages/_document.tsx [hunk 2]
```diff
@@ -1,13 +1,18 @@
-import { Html, Head, Main, NextScript } from 'next/document'
+import { Html, Head, Main, NextScript } from "next/document";
 
 export default function Document() {
   return (
     <Html lang="en">
```
[the remark this answers is not in the document]
Comment (star82620) [reply pages/_document.tsx:5]: 修正為 <HTML lang="zh-Hant-TW">

File: styles/globals.css (hunk 1 above)
Comment (star82620) [reply styles/globals.css:39]: @tailwind base 中已有 reset 動作

File: pages/_document.tsx [hunk 3]
```diff
@@ -1,13 +1,18 @@
-import { Html, Head, Main, NextScript } from 'next/document'
+import { Html, Head, Main, NextScript } from "next/document";
 
 export default function Document() {
   return (
-    <Html lang="en">
+    <Html lang="zh-Hant-TW">
```
Comment (seonkuraito) [inline pages/_document.tsx:5]: `zh-tw`

**Review by seonkuraito — APPROVED** [review]

**Pull request merged**